### ISSUE TITLE
[Bug] chia plots check

### ISSUE THREAD

**Issue by Ntohpn** [Chia-Network/chia-blockchain#16137]
### What happened?

![image](https://github.com/Chia-Network/chia-blockchain/assets/97425684/54f1d698-3874-48c5-9696-b93cecbb7a74)
"Has health check been abandoned in the era of compressed images?"
"Has health check been abandoned in the era of compressed images?"
"Has health check been abandoned in the era of compressed images?"

### Version

2.0RC6

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
C:\Chia\daemon>chia plots check -g B:\
2023-08-23T15:05:15.121  chia.plotting.check_plots        : INFO     Loading plots in config.yaml using plot_manager loading code (parallel read: True)

2023-08-23T15:05:15.330  chia.plotting.cache              : INFO     Loaded 5780 bytes of cached data
2023-08-23T15:05:15.331  chia.plotting.cache              : INFO     Parsed 9 cache entries in 0.00s
2023-08-23T15:05:15.398  chia.plotting.check_plots        : INFO     event: started, loaded 0 plots, 8 remaining
2023-08-23T15:05:15.398  chia.plotting.manager            : INFO     Only loading plots that contain "B:\" in the file or directory name
2023-08-23T15:05:15.399  chia.plotting.manager            : WARNING  Plot B:\plot-k32-c05-2023-08-23-13-18-6dc18cb79b52ac01d0ee79fefa61e47b793e2ef1f0e19aaadaf87ba0e4941bba.plot has a farmer public key that is not in the farmer's pk list.
2023-08-23T15:05:15.400  chia.plotting.manager            : WARNING  Plot B:\plot-k32-c05-2023-08-23-13-24-2e21de611dabf7e018bbd96815d14b6e8e23cdb537735fecbfc8a7c8df67bdf4.plot has a farmer public key that is not in the farmer's pk list.
2023-08-23T15:05:15.401  chia.plotting.manager            : WARNING  Plot B:\plot-k32-c05-2023-08-23-13-51-44304ab8ed48151780ae6a9173affc81817da4aa28bba209fe6e116925df5e63.plot has a farmer public key that is not in the farmer's pk list.
2023-08-23T15:05:15.403  chia.plotting.manager            : WARNING  Plot B:\plot-k32-c05-2023-08-23-13-56-c286ab5c10a97827a88367166c0f087e81a5632af6e833b9dfd2d094900ab2cc.plot has a farmer public key that is not in the farmer's pk list.
2023-08-23T15:05:15.403  chia.plotting.manager            : WARNING  Plot B:\plot-k32-c05-2023-08-23-14-13-dff925e8b271f8cbeaddbfc4ca84fc0de405b276b11b12238724714c3b238b0c.plot has a farmer public key that is not in the farmer's pk list.
2023-08-23T15:05:15.404  chia.plotting.manager            : WARNING  Plot B:\plot-k32-c05-2023-08-23-14-08-a90fb58b65ea07a0eb3f51460330ad0524a91f9cf7d6bb8065cdb4371f54ddb7.plot has a farmer public key that is not in the farmer's pk list.
2023-08-23T15:05:15.404  chia.plotting.manager            : WARNING  Plot B:\plot-k32-c05-2023-08-23-14-19-2c35ab5c3a21ec9e7fc7dfab936223054c37466aa2a691127a56b0aa62f0c8f6.plot has a farmer public key that is not in the farmer's pk list.
2023-08-23T15:05:15.404  chia.plotting.manager            : WARNING  Plot B:\plot-k32-c07-2023-08-23-13-31-7bbe87486d0db0f9f6a32b9b2f7f59552f6240714d97aa1075f2c2c3ee77e463.plot has a farmer public key that is not in the farmer's pk list.
2023-08-23T15:05:15.407  chia.plotting.check_plots        : INFO     event: batch_processed, loaded 8 plots, 0 remaining2023-08-23T15:05:15.408  chia.plotting.check_plots        : INFO     event: done, loaded 8 plots, 0 remaining
2023-08-23T15:05:16.411  chia.plotting.check_plots        : INFO
2023-08-23T15:05:16.412  chia.plotting.check_plots        : INFO
2023-08-23T15:05:16.413  chia.plotting.check_plots        : INFO     Starting to test each plot with 30 challenges each

2023-08-23T15:05:16.430  chia.plotting.check_plots        : INFO     Testing plot B:\plot-k32-c05-2023-08-23-13-18-6dc18cb79b52ac01d0ee79fefa61e47b793e2ef1f0e19aaadaf87ba0e4941bba.plot k=32
2023-08-23T15:05:16.431  chia.plotting.check_plots        : INFO        Pool contract address:  xch1d72f2d4r3yjx24ewe2c0qy9fx5ta4un4pcnm6t9p9vl2s98j3flqkua56x
2023-08-23T15:05:16.463  chia.plotting.check_plots        : INFO        Farmer public key:      ad0b7b5b9e82ff895ff9b82e4f0e282f454bd823bc41e01228e126be4259e80fe170fc29bb903af9a48c0444ee34afbf
2023-08-23T15:05:16.464  chia.plotting.check_plots        : INFO        Local sk:               <PrivateKey 3de5511b35a6272a71fc7ad7e886e56e8407ab275d230c3bf59b2b4e86b25694>
2023-08-23T15:05:16.466  chia.plotting.check_plots        : INFO     Testing plot B:\plot-k32-c05-2023-08-23-13-24-2e21de611dabf7e018bbd96815d14b6e8e23cdb537735fecbfc8a7c8df67bdf4.plot k=32
2023-08-23T15:05:16.466  chia.plotting.check_plots        : INFO        Pool contract address:  xch1d72f2d4r3yjx24ewe2c0qy9fx5ta4un4pcnm6t9p9vl2s98j3flqkua56x
2023-08-23T15:05:16.467  chia.plotting.check_plots        : INFO        Farmer public key:      ad0b7b5b9e82ff895ff9b82e4f0e282f454bd823bc41e01228e126be4259e80fe170fc29bb903af9a48c0444ee34afbf
2023-08-23T15:05:16.467  chia.plotting.check_plots        : INFO        Local sk:               <PrivateKey 33a8a8f49d10b9d844e4441a8f50bd4cbcac8a8ef4e7557787a0d2368b66685a>
2023-08-23T15:05:16.468  chia.plotting.check_plots        : INFO     Testing plot B:\plot-k32-c05-2023-08-23-13-51-44304ab8ed48151780ae6a9173affc81817da4aa28bba209fe6e116925df5e63.plot k=32
2023-08-23T15:05:16.468  chia.plotting.check_plots        : INFO        Pool contract address:  xch1d72f2d4r3yjx24ewe2c0qy9fx5ta4un4pcnm6t9p9vl2s98j3flqkua56x
2023-08-23T15:05:16.469  chia.plotting.check_plots        : INFO        Farmer public key:      ad0b7b5b9e82ff895ff9b82e4f0e282f454bd823bc41e01228e126be4259e80fe170fc29bb903af9a48c0444ee34afbf
2023-08-23T15:05:16.469  chia.plotting.check_plots        : INFO        Local sk:               <PrivateKey 22c7449d08767b265cb160abe7a43e4786aabc8bcb3e494c7b8e975417822fa4>
2023-08-23T15:05:16.470  chia.plotting.check_plots        : INFO     Testing plot B:\plot-k32-c05-2023-08-23-13-56-c286ab5c10a97827a88367166c0f087e81a5632af6e833b9dfd2d094900ab2cc.plot k=32
2023-08-23T15:05:16.470  chia.plotting.check_plots        : INFO        Pool contract address:  xch1d72f2d4r3yjx24ewe2c0qy9fx5ta4un4pcnm6t9p9vl2s98j3flqkua56x
2023-08-23T15:05:16.471  chia.plotting.check_plots        : INFO        Farmer public key:      ad0b7b5b9e82ff895ff9b82e4f0e282f454bd823bc41e01228e126be4259e80fe170fc29bb903af9a48c0444ee34afbf
2023-08-23T15:05:16.471  chia.plotting.check_plots        : INFO        Local sk:               <PrivateKey 33f039d82434dd3bc81f3667f5e8f70d69dad9a7f2cf672442a967248d6b4456>
2023-08-23T15:05:16.484  chia.plotting.check_plots        : ERROR       Proofs 0 / 30, 0.0 Filepath: B:\plot-k32-c05-2023-08-23-13-18-6dc18cb79b52ac01d0ee79fefa61e47b793e2ef1f0e19aaadaf87ba0e4941bba.plot
2023-08-23T15:05:16.490  chia.plotting.check_plots        : ERROR       Proofs 0 / 30, 0.0 Filepath: B:\plot-k32-c05-2023-08-23-13-24-2e21de611dabf7e018bbd96815d14b6e8e23cdb537735fecbfc8a7c8df67bdf4.plot
2023-08-23T15:05:16.491  chia.plotting.check_plots        : INFO     Testing plot B:\plot-k32-c05-2023-08-23-14-08-a90fb58b65ea07a0eb3f51460330ad0524a91f9cf7d6bb8065cdb4371f54ddb7.plot k=32
2023-08-23T15:05:16.492  chia.plotting.check_plots        : INFO        Pool contract address:  xch1d72f2d4r3yjx24ewe2c0qy9fx5ta4un4pcnm6t9p9vl2s98j3flqkua56x
2023-08-23T15:05:16.492  chia.plotting.check_plots        : INFO        Farmer public key:      ad0b7b5b9e82ff895ff9b82e4f0e282f454bd823bc41e01228e126be4259e80fe170fc29bb903af9a48c0444ee34afbf
2023-08-23T15:05:16.493  chia.plotting.check_plots        : INFO        Local sk:               <PrivateKey 48a129c412df3a5a2cca55eaee28f512895c24813cd362a29b097bdc8163c2b3>
2023-08-23T15:05:16.494  chia.plotting.check_plots        : ERROR       Proofs 0 / 30, 0.0 Filepath: B:\plot-k32-c05-2023-08-23-13-51-44304ab8ed48151780ae6a9173affc81817da4aa28bba209fe6e116925df5e63.plot
2023-08-23T15:05:16.498  chia.plotting.check_plots        : INFO     Testing plot B:\plot-k32-c05-2023-08-23-14-13-dff925e8b271f8cbeaddbfc4ca84fc0de405b276b11b12238724714c3b238b0c.plot k=32
2023-08-23T15:05:16.498  chia.plotting.check_plots        : INFO        Pool contract address:  xch1d72f2d4r3yjx24ewe2c0qy9fx5ta4un4pcnm6t9p9vl2s98j3flqkua56x
2023-08-23T15:05:16.499  chia.plotting.check_plots        : INFO        Farmer public key:      ad0b7b5b9e82ff895ff9b82e4f0e282f454bd823bc41e01228e126be4259e80fe170fc29bb903af9a48c0444ee34afbf
2023-08-23T15:05:16.499  chia.plotting.check_plots        : INFO        Local sk:               <PrivateKey 1cea27239e2afc2274184ac2b2cda033b260146e807700fd45d7b51fb0e6055d>
2023-08-23T15:05:16.501  chia.plotting.check_plots        : ERROR       Proofs 0 / 30, 0.0 Filepath: B:\plot-k32-c05-2023-08-23-14-08-a90fb58b65ea07a0eb3f51460330ad0524a91f9cf7d6bb8065cdb4371f54ddb7.plot
2023-08-23T15:05:16.502  chia.plotting.check_plots        : INFO     Testing plot B:\plot-k32-c05-2023-08-23-14-19-2c35ab5c3a21ec9e7fc7dfab936223054c37466aa2a691127a56b0aa62f0c8f6.plot k=32
2023-08-23T15:05:16.503  chia.plotting.check_plots        : INFO        Pool contract address:  xch1d72f2d4r3yjx24ewe2c0qy9fx5ta4un4pcnm6t9p9vl2s98j3flqkua56x
2023-08-23T15:05:16.503  chia.plotting.check_plots        : INFO        Farmer public key:      ad0b7b5b9e82ff895ff9b82e4f0e282f454bd823bc41e01228e126be4259e80fe170fc29bb903af9a48c0444ee34afbf
2023-08-23T15:05:16.504  chia.plotting.check_plots        : INFO        Local sk:               <PrivateKey 7384a4fff152a7c1dcca2396ac6dd4cc3fc97b132c5d6aa9d835b97bb8ef9492>
2023-08-23T15:05:16.505  chia.plotting.check_plots        : ERROR       Proofs 0 / 30, 0.0 Filepath: B:\plot-k32-c05-2023-08-23-13-56-c286ab5c10a97827a88367166c0f087e81a5632af6e833b9dfd2d094900ab2cc.plot
2023-08-23T15:05:16.506  chia.plotting.check_plots        : ERROR       Proofs 0 / 30, 0.0 Filepath: B:\plot-k32-c05-2023-08-23-14-13-dff925e8b271f8cbeaddbfc4ca84fc0de405b276b11b12238724714c3b238b0c.plot
2023-08-23T15:05:16.508  chia.plotting.check_plots        : ERROR       Proofs 0 / 30, 0.0 Filepath: B:\plot-k32-c05-2023-08-23-14-19-2c35ab5c3a21ec9e7fc7dfab936223054c37466aa2a691127a56b0aa62f0c8f6.plot
2023-08-23T15:05:16.509  chia.plotting.check_plots        : INFO     Testing plot B:\plot-k32-c07-2023-08-23-13-31-7bbe87486d0db0f9f6a32b9b2f7f59552f6240714d97aa1075f2c2c3ee77e463.plot k=32
2023-08-23T15:05:16.510  chia.plotting.check_plots        : INFO        Pool contract address:  xch1d72f2d4r3yjx24ewe2c0qy9fx5ta4un4pcnm6t9p9vl2s98j3flqkua56x
2023-08-23T15:05:16.510  chia.plotting.check_plots        : INFO        Farmer public key:      ad0b7b5b9e82ff895ff9b82e4f0e282f454bd823bc41e01228e126be4259e80fe170fc29bb903af9a48c0444ee34afbf
2023-08-23T15:05:16.510  chia.plotting.check_plots        : INFO        Local sk:               <PrivateKey 51716090b8831093ce66475193477df979e62cde935b14a00441f2ba03517389>
2023-08-23T15:05:16.516  chia.plotting.check_plots        : ERROR       Proofs 0 / 30, 0.0 Filepath: B:\plot-k32-c07-2023-08-23-13-31-7bbe87486d0db0f9f6a32b9b2f7f59552f6240714d97aa1075f2c2c3ee77e463.plot
2023-08-23T15:05:16.517  chia.plotting.check_plots        : INFO
2023-08-23T15:05:16.518  chia.plotting.check_plots        : INFO
2023-08-23T15:05:16.518  chia.plotting.check_plots        : INFO     Summary
2023-08-23T15:05:16.519  chia.plotting.check_plots        : INFO     Found 0 valid plots, total size 0.00000 TiB
2023-08-23T15:05:16.519  chia.plotting.check_plots        : WARNING  8 invalid plots found:
2023-08-23T15:05:16.520  chia.plotting.check_plots        : WARNING      8 bad plots:
2023-08-23T15:05:16.520  chia.plotting.check_plots        : WARNING  B:\plot-k32-c05-2023-08-23-13-18-6dc18cb79b52ac01d0ee79fefa61e47b793e2ef1f0e19aaadaf87ba0e4941bba.plot
2023-08-23T15:05:16.521  chia.plotting.check_plots        : WARNING  B:\plot-k32-c05-2023-08-23-13-24-2e21de611dabf7e018bbd96815d14b6e8e23cdb537735fecbfc8a7c8df67bdf4.plot
2023-08-23T15:05:16.521  chia.plotting.check_plots        : WARNING  B:\plot-k32-c05-2023-08-23-13-51-44304ab8ed48151780ae6a9173affc81817da4aa28bba209fe6e116925df5e63.plot
2023-08-23T15:05:16.522  chia.plotting.check_plots        : WARNING  B:\plot-k32-c05-2023-08-23-14-08-a90fb58b65ea07a0eb3f51460330ad0524a91f9cf7d6bb8065cdb4371f54ddb7.plot
2023-08-23T15:05:16.522  chia.plotting.check_plots        : WARNING  B:\plot-k32-c05-2023-08-23-13-56-c286ab5c10a97827a88367166c0f087e81a5632af6e833b9dfd2d094900ab2cc.plot
2023-08-23T15:05:16.523  chia.plotting.check_plots        : WARNING  B:\plot-k32-c05-2023-08-23-14-13-dff925e8b271f8cbeaddbfc4ca84fc0de405b276b11b12238724714c3b238b0c.plot
2023-08-23T15:05:16.523  chia.plotting.check_plots        : WARNING  B:\plot-k32-c05-2023-08-23-14-19-2c35ab5c3a21ec9e7fc7dfab936223054c37466aa2a691127a56b0aa62f0c8f6.plot
2023-08-23T15:05:16.524  chia.plotting.check_plots        : WARNING  B:\plot-k32-c07-2023-08-23-13-31-7bbe87486d0db0f9f6a32b9b2f7f59552f6240714d97aa1075f2c2c3ee77e463.plot
```

**Comment by wjblanke** [comment]
What OS version are you plotting on? Are the plots able to produce valid partials?

**Comment by jmhands** [comment]
please create plots with the release version of bladebit
https://github.com/Chia-Network/bladebit/releases/tag/v3.0.0
and let us know if the check works

**Comment by keithxiaoa** [comment]
Chia's official plot such like 'chia plotters bladebit cudaplot ', this program create plots can't pass 'chia plots check -n 30 -g xxxpath' 

use https://github.com/Chia-Network/bladebit/releases/tag/v3.0.0
create plots such as '/your_download_dir/bladebit_cuda -t 10 -n 191 -f xxx -c xxx -z 7 cudaplot /xxx/xxx/xxxx', can pass 'chia plots check -n 30 -g xxxpath'

**Comment by BrandtH22** [comment]
> 很明显我尝试了但是不可以

When running the command did you add in your own farmer and contract information replacing the "xxx" in `-f xxx -c xxx` ?

**Comment by wjblanke** [comment]
Windows 10 does not produce valid plots in the 2.0 release. Are you using windows 10?

**Comment by emlowe** [comment]
Closing issue.

Windows 10 does not make correct plots in 2.0.0 or 2.0.1
There was a bug in 2.0.0 on Linux that resulted in bad plots when using `chia plotters` or the GUI - this is resolved in 2.0.1
